### PR TITLE
update node-xmpp-client to 3.0

### PR DESCRIPTION
**Pull request by sonnyp**
I'm the maintainer of node-xmpp, native dependencies are not required anymore.

### CODE REVIEW

File: lib/simple-xmpp.js [hunk 1]
```diff
@@ -46,7 +47,6 @@ function SimpleXMPP() {
     //setting status here
     this.STATUS = STATUS;
     var self = this;
-    this.Element = xmpp.Element;
```
Comment (sonnyp) [inline lib/simple-xmpp.js]: moved to SimpleXMPP prototype

I recommend to move other commondmethods/properties on the prototype as well

File: readme.md [hunk 2]
```diff
@@ -2,10 +2,6 @@ Node Simple XMPP
 ================
 Simple High Level NodeJS XMPP Library
 
-Dependencies
-------------
-	sudo apt-get install libexpat1 libexpat1-dev libicu-dev
-
```
Comment (sonnyp) [inline readme.md]: those are not required anymore

**Comment by silverbucket** [comment]
Nice, thanks @sonnyp - I imagine you did some testing? we don't have any unit tests so it's pretty much up to manual testing.

**Comment by sonnyp** [comment]
@silverbucket I didn't

**Pull request merged**